config/sync/system.performance.yml

```diff
@@ -2,15 +2,15 @@ cache:
   page:
     max_age: 0
 css:
-  preprocess: true
+  preprocess: false
```

Comment:
Don't leave these turned off in the config. If you need these settings to remain false on your local, append these two lines to your `web/sites/default/settings.local.php`:

```php
$config['system.performance']['css']['preprocess'] = FALSE;
$config['system.performance']['js']['preprocess'] = FALSE;
```

Reenable them in Configuration -> Performance and reexport the config.